mocks/Cargo.toml

```
@@ -1,11 +1,12 @@
 [package]
 name = "mocks"
 version = "0.0.2"
-authors = ["Request Labs"]
+authors = ["Request Finance"]
```

Please consider adding Request Network as an author.

```suggestion
authors = ["Request Finance", "Request Network"]
```